frontend/src/components/profile/my-tickets.tsx

```diff
@@ -0,0 +1,228 @@
+/** @jsxRuntime classic */
+
+/** @jsx jsx */
+import { useState, useCallback } from "react";
+import { FormattedMessage } from "react-intl";
+import { Box, Heading, jsx } from "theme-ui";
+
+import { useRouter } from "next/router";
+
+import { Alert } from "~/components/alert";
+import { Button } from "~/components/button/button";
+import { Modal } from "~/components/modal";
+import { Table } from "~/components/table";
+import { QuestionsSection } from "~/components/tickets-page/questions-section";
+import { useTranslatedMessage } from "~/helpers/use-translated-message";
+import { AttendeeTicket, useUpdateTicketMutation } from "~/types";
+
+import { ProductState } from "../tickets-page/types";
+import { Link } from "~/components/link";
+
+type Props = {
+  tickets?: AttendeeTicket[];
+};
+
+export const MyTickets: React.FC<Props> = ({ tickets }) => {
```

Comment:
you don't need React.FC if you don't use `children`



```suggestion
export const MyTickets = ({ tickets }: Props) => {
```